### ISSUE TITLE
vulkan samples not building

### ISSUE THREAD

**Issue by udnaan** [NVIDIA/cuda-samples#50]
## What
Vulkan samples do not build

## Where
Debian buster (10.5) X86_64

## Why
findvulkan.mk has hardcoded paths for each distro. Unfortunately, debian is not one of the included distro and since the makefile fails to match a case, it declares libX11.so as missing. 

## Solution
`/usr/sbin/ldconfig -p | grep -i libx11` is a more reliable way to search for libraries on any modern linux system that cuda supports.

**Comment by mdoijade** [comment]
that is correct Debian OS support needs to be added, thanks for the suggestion regarding ldconfig.

**Comment by qhaas** [comment]
Had similar issues in [EL7](https://github.com/NVIDIA/cuda-samples/issues/33)

**Comment by rwarmstr** [comment]
Given the age of this issue will close as resolved - also update to CMake should help here.
